commands/games/roulette.js

```diff
@@ -0,0 +1,89 @@
+const { Command } = require('discord.js-commando');
+const stripIndents = require('common-tags').stripIndents;
+
+const Currency = require('../../currency/Currency');
+const Roulette = require('../../Roulette');
+
+const currency = new Currency();
+
+module.exports = class RouletteCommand extends Command {
+	constructor(client) {
+		super(client, {
+			name: 'roulette',
+			aliases: ['roulette', 'bet'],
+			group: 'games',
+			memberName: 'roulette',
+			description: 'Play a game of russian roulette for donuts!',
+			details: 'Play a game of russian roulette for donuts.',
+			throttling: {
+				duration: 30,
```

Comment:
Keep those 30sec? Might wanna set that higher? Not sure, think about how long one round usually takes and take into account how long it needs to cooldown. Cooldown starts at command useage.